src/components/content/entity.tsx

```diff
@@ -59,7 +63,11 @@ export function Entity({ data }: EntityProps) {
       {renderStyledH1()}
       {renderUserTools({ aboveContent: true })}
       <div className="min-h-1/4" key={data.id}>
-        {data.content && renderContent(data.content)}
+        {isMockupCoursePage ? (
```

Comment:
It would be great not having to change this file just for a mock, any ideas?

Reply:
If I remember correctly last time we did not merge it into staging but left it on a branch, right?

Reply:
> If I remember correctly last time we did not merge it into staging but left it on a branch, right?

yep, that's fine

Reply:
ok, in that case, please close the PR if you don't have anything to change.